### ISSUE TITLE
LG V30 is not yet supported

### ISSUE THREAD

**Issue by 01098996** [google-ar/arcore-android-sdk#320]
1.Call _ArCamera_getViewMatrix_ result  always  identity matrix.
2.Call _ArFrame_acquireCameraImage_ always return AR_ERROR_NOT_YET_AVAILABLE.

Below is the log:

```
04-18 17:36:45.157 1811-2761/com.meitu.arcoredemo E/native: status.cc:175 ArStatusErrorSpace::AR_ERROR_NOT_YET_AVAILABLE: 
04-18 17:36:45.189 1811-2761/com.meitu.arcoredemo E/native: status.cc:175 ArStatusErrorSpace::AR_ERROR_NOT_YET_AVAILABLE: 
04-18 17:36:45.223 1811-2761/com.meitu.arcoredemo E/native: status.cc:175 ArStatusErrorSpace::AR_ERROR_NOT_YET_AVAILABLE: 
04-18 17:36:45.255 1811-2761/com.meitu.arcoredemo E/native: status.cc:175 ArStatusErrorSpace::AR_ERROR_NOT_YET_AVAILABLE: 
04-18 17:36:45.288 1811-2761/com.meitu.arcoredemo E/native: status.cc:175 ArStatusErrorSpace::AR_ERROR_NOT_YET_AVAILABLE: 
04-18 17:36:45.322 1811-2761/com.meitu.arcoredemo E/native: status.cc:175 ArStatusErrorSpace::AR_ERROR_NOT_YET_AVAILABLE: 
04-18 17:36:45.357 1811-2761/com.meitu.arcoredemo E/native: status.cc:175 ArStatusErrorSpace::AR_ERROR_NOT_YET_AVAILABLE: 
04-18 17:36:45.390 1811-2761/com.meitu.arcoredemo E/native: status.cc:175 ArStatusErrorSpace::AR_ERROR_NOT_YET_AVAILABLE: 
04-18 17:36:45.423 1811-2761/com.meitu.arcoredemo E/native: status.cc:175 ArStatusErrorSpace::AR_ERROR_NOT_YET_AVAILABLE: 
04-18 17:36:45.458 1811-2761/com.meitu.arcoredemo E/native: status.cc:175 ArStatusErrorSpace::AR_ERROR_NOT_YET_AVAILABLE: 
04-18 17:36:45.491 1811-2761/com.meitu.arcoredemo E/native: status.cc:175 ArStatusErrorSpace::AR_ERROR_NOT_YET_AVAILABLE: 
04-18 17:36:45.497 1146-2842/? E/LG-3A-AWB: lg_awb_always_gain_logging: main log - final output_gain(2.174281/1.000000/1.608720), out_cct(5231), out_type(1)
04-18 17:36:45.526 1811-2761/com.meitu.arcoredemo E/native: status.cc:175 ArStatusErrorSpace::AR_ERROR_NOT_YET_AVAILABLE: 
04-18 17:36:45.560 1811-2761/com.meitu.arcoredemo E/native: status.cc:175 ArStatusErrorSpace::AR_ERROR_NOT_YET_AVAILABLE: 
04-18 17:36:45.597 1811-2761/com.meitu.arcoredemo E/native: status.cc:175 ArStatusErrorSpace::AR_ERROR_NOT_YET_AVAILABLE: 
04-18 17:36:45.624 1811-2761/com.meitu.arcoredemo E/native: status.cc:175 ArStatusErrorSpace::AR_ERROR_NOT_YET_AVAILABLE: 
04-18 17:36:45.659 1811-2761/com.meitu.arcoredemo E/native: status.cc:175 ArStatusErrorSpace::AR_ERROR_NOT_YET_AVAILABLE: 
04-18 17:36:45.693 1811-2761/com.meitu.arcoredemo E/native: status.cc:175 ArStatusErrorSpace::AR_ERROR_NOT_YET_AVAILABLE: 
04-18 17:36:45.728 1811-2761/com.meitu.arcoredemo E/native: status.cc:175 ArStatusErrorSpace::AR_ERROR_NOT_YET_AVAILABLE: 
04-18 17:36:45.759 1811-2761/com.meitu.arcoredemo E/native: status.cc:175 ArStatusErrorSpace::AR_ERROR_NOT_YET_AVAILABLE: 
04-18 17:36:45.793 1811-2761/com.meitu.arcoredemo E/native: status.cc:175 ArStatusErrorSpace::AR_ERROR_NOT_YET_AVAILABLE: 
04-18 17:36:45.822 1146-2842/? E/LG-3A-AWB: lg_awb_always_gain_logging: main log - final output_gain(2.180208/1.000000/1.604891), out_cct(5253), out_type(1)
04-18 17:36:45.827 1811-2761/com.meitu.arcoredemo E/native: status.cc:175 ArStatusErrorSpace::AR_ERROR_NOT_YET_AVAILABLE: 
04-18 17:36:45.858 1811-2761/com.meitu.arcoredemo E/native: status.cc:175 ArStatusErrorSpace::AR_ERROR_NOT_YET_AVAILABLE: 
04-18 17:36:45.891 1811-2761/com.meitu.arcoredemo E/native: status.cc:175 ArStatusErrorSpace::AR_ERROR_NOT_YET_AVAILABLE: 
04-18 17:36:45.922 1811-2761/com.meitu.arcoredemo E/native: status.cc:175 ArStatusErrorSpace::AR_ERROR_NOT_YET_AVAILABLE: 
04-18 17:36:45.957 1811-2761/com.meitu.arcoredemo E/native: status.cc:175 ArStatusErrorSpace::AR_ERROR_NOT_YET_AVAILABLE: 
04-18 17:36:45.991 1811-2761/com.meitu.arcoredemo E/native: status.cc:175 ArStatusErrorSpace::AR_ERROR_NOT_YET_AVAILABLE: 
04-18 17:36:46.025 1811-2761/com.meitu.arcoredemo E/native: status.cc:175 ArStatusErrorSpace::AR_ERROR_NOT_YET_AVAILABLE: 
04-18 17:36:46.059 1811-2761/com.meitu.arcoredemo E/native: status.cc:175 ArStatusErrorSpace::AR_ERROR_NOT_YET_AVAILABLE: 
04-18 17:36:46.094 1811-2761/com.meitu.arcoredemo E/native: status.cc:175 ArStatusErrorSpace::AR_ERROR_NOT_YET_AVAILABLE: 
04-18 17:36:46.127 1811-2761/com.meitu.arcoredemo E/native: status.cc:175 ArStatusErrorSpace::AR_ERROR_NOT_YET_AVAILABLE: 
```

**Comment by inio** [comment]
The LG H930 is not a [supported device](https://developers.google.com/ar/discover/#supported_devices).

**Comment by inio** [comment]
Duplicate of #89

**Comment by 01098996** [comment]
https://developers.google.com/ar/discover/?hl=zh-cn 
Indicates that LG H930 is supported @inio

**Comment by inio** [comment]
Oops!  I didn't realize the H930 was a model of the V30.

We encountered a driver issue very late in enabling the V30.  It will be supported soon.

**Comment by 01098996** [comment]
Please support as soon as possible.Thanks!

**Comment by mxxcon** [comment]
What does it mean that you removed all the issue tags? It won't be fixed and there's no work being done to add support for V30?

**Comment by justinmyersm** [comment]
I'd also like to know why the V30 was removed from the list of supported devices. It used to state that you needed Oreo, now the device is not listed at all. Please help!

**Comment by inio** [comment]
I removed the "fixed in upcoming release" tag in preparation for the 1.2 release to avoid confusion on the meanings of "upcoming" and "next".  We're still blocked on a driver bug on this device.  Calibrations are in 1.2 and it *may* work if side-loaded, but we cannot fully support the device and enable it on Play Store because there are too many devices in the field that don't have the driver update yet.

**Comment by widovanheemstra** [comment]
When will Huawei mate 10 pro be supported? It should be technical feasible to support AR.

**Comment by inio** [comment]
@widovanheemstra Please post your request in #89.  We cannot comment on upcoming support before it is announced.

**Comment by forcewake** [comment]
Hello @inio 
Do you have any updates on this issue?

**Comment by inio** [comment]
@forcewake Nope.

**Comment by inio** [comment]
This should be fixed!  Users with V30 devices should now be able to see ARCore and AR-Required apps in the Play Store.  If you experience poor functioning or tracking oddities, please make sure you don't have any pending security/system updates before reporting.
